### ISSUE TITLE
Ability to delete my own upload after the allotted time

### ISSUE THREAD

**Issue by PFCKrutonium** [gitterHQ/gitter#975]
Not comments, that is fine, but I need the ability to delete a sensitive file if it happens to be uploaded.

**Comment by MadLittleMods** [comment]
Duplicate of https://github.com/gitterHQ/gitter/issues/332

**Comment by PFCKrutonium** [comment]
I'm sorry, this this is a Serious issue that you guys should not be turning a blind eye to.

**Comment by MadLittleMods** [comment]
@PFCKrutonium I think that a confirmation type dialog to review before posting would be great. But keeping the longevity and expectation that links will live forever is an important aspect to make sure messages can live on in the future (dead links are no fun).

**Comment by PFCKrutonium** [comment]
While I agree with what you are getting at, there are times when being able to delete a file is very important. Lets say for example, I upload a text file, and only later realize I accidentally left the password to [expensive thing here]. While yes, I can just change the password, and I would, having that file exist, with that file, can make future access to any account I have, easier due to knowing how I build a password.

 And what about copyright infringing files? Or files that were never intended to be shared? There _needs_ to be a way to remove the files after the fact. Or at least make the window for removal longer for files, as well as your proposed confirm box. 12 Hours would be a reasonable time for this - a file can contain a lot more info than a text based message, and you may not realize until it is too late.

Basically what I am trying to say, is this needs to happen.
